docs/Asset/Api/EmailTemplatesApi.md

```diff
@@ -120,7 +124,7 @@ No authorization required
 [[Back to top]](#) [[Back to API list]](../../README.md#documentation-for-api-endpoints) [[Back to Model list]](../../README.md#documentation-for-models) [[Back to README]](../../README.md)
 
 # **createEmailTemplateUsingPOST**
-> \NecLimDul\MarketoRest\Asset\Model\ResponseOfEmailTemplateResponse createEmailTemplateUsingPOST($create_email_template_request)
+> \NecLimDul\MarketoRest\Asset\Model\ResponseOfEmailTemplateResponse createEmailTemplateUsingPOST($name, $folder, $content, $description)
```

Comment:
I don't know that these changes are ideal. The new generated code is less like the swagger docs on the Marketo site and even though its a bit more work I would much rather have the object to match the documentation.

This seems to be a feature of some sort in OpenApiTools. Hopefully there's a way to work around it and get the more accurate code.